src/components/disclosure/disclosure.module.css

```diff
@@ -0,0 +1,98 @@
+.root {
+  background-color: white;
+  border-radius: 6px;
+  width: 100%;
+}
+
+.root-expanded {
+  composes: hds-surface-mid from global;
+}
+
+.button {
+  composes: hds-surface-mid from global;
+  align-items: center;
+  background-color: transparent;
+  border-radius: 6px;
+  border: none;
+  cursor: pointer;
+  display: flex;
+  padding: 16px;
+  text-align: left;
+  width: 100%;
+
+  & svg {
+    color: var(--token-color-neutral-foreground-faint);
+
+    /* Only enable animation if query is supported and value is no-preference */
+    @media (prefers-reduced-motion: no-preference) {
+      transition: transform 0.2s;
+    }
+  }
+
+  /* aka Disclosure is open */
+  &[aria-expanded='true'] {
+    box-shadow: none;
+
+    & svg {
+      transform: rotate(90deg);
+    }
+  }
+
+  /* aka Disclosure is closed */
+  &[aria-expanded='false'] {
+    &:hover {
+      box-shadow: var(--token-surface-high-box-shadow);
+      cursor: pointer;
+    }
+  }
```

Comment:
nit ✨ : consider adding a transition here (same as we use on `CardLink`) for a little extra polish and consistency.

Reply:
I probably should have noted this in the Anything else? section, I tried some transitions but they weren't behaving quite right! I was able to get the transition to work correctly when opening a Disclosure, but it wasn't working when they closed. 🤔

Reply:
I think a follow-up PR might be good for adding that code back in and then passing the branch along to the team for one of our ✨ CSS wizards ✨ to take a look at.

Reply:
Would that be a transition for the height? What about just the box-shadow transition on hover of a collapsed `Disclosure`?

Reply:
Added in https://github.com/hashicorp/dev-portal/pull/96/commits/2154a6f7628345d98034723a4cabbd2aaa18c44c